README.md

```diff
@@ -1,12 +1,33 @@
-[![Build Status](https://travis-ci.org/jump-dev/MosekTools.jl.svg?branch=master)](https://travis-ci.org/jump-dev/MosekTools.jl)
+# MosekTools.jl
 
-``MosekTools`` is the
+[MosekTools.jl](https://github.com/jump-dev/MosekTools.jl) is the
 [MathOptInterface.jl](https://github.com/jump-dev/MathOptInterface.jl)
-implementation for the MOSEK solver. The low-level solver API for MOSEK is
-found in the package [Mosek.jl](https://github.com/MOSEK/Mosek.jl).
+implementation for the MOSEK solver.
+
+The low-level solver API for MOSEK is found in the package [Mosek.jl](https://github.com/MOSEK/Mosek.jl).
+
+## Affiliation
+
+MosekTools.jl is maintained by the JuMP community and is not officially
+supported by MOSEK. However, Mosek.jl _is_ an officially supported product of
+MOSEK.
+
+## License
+
+`MosekTools.jl` is licensed under the [MIT License](https://github.com/jump-dev/MosekTools.jl/blob/master/LICENSE.md).
+
+The underlying solver is a closed-source commercial product for which you must
+[purchase a license](https://www.mosek.com).
+
+## Innstallation
```

Comment:
Typo

Reply:
Yeah. I'll go through them all carefully once I've updated all the repos (there's a lot to do) and we have a single build of the documentation. The linter will also run on all the READMEs, so that'll catch typos like this.